docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
       file: docker-compose-base.yml
       service: base
     env_file:
-      - conf/development.env
```

Comment:
This should be set to development.env - default.env is added in docker-compose-base.

Maybe development.env.example should just be committed as development.env though...